pinot-transport/src/main/java/com/linkedin/pinot/transport/scattergather/ScatterGatherImpl.java

```diff
@@ -141,10 +160,17 @@ public ScatterGatherImpl(KeyedPool<ServerInstance, NettyClientConnection> pool,
           new ArrayList<KeyedFuture<ServerInstance, ByteBuf>>();
       for (SingleRequestHandler h : handlers) {
         responseFutures.add(h.getResponseFuture());
-        final String server = h.getServer().toString();
-        scatterGatherStats.setSendStartTimeMillis(server, h.getConnStartTimeMillis());
-        scatterGatherStats.setConnStartTimeMillis(server, h.getStartDelayMillis());
-        scatterGatherStats.setSendCompletionTimeMillis(server, h.getSendCompletionTimeMillis());
+        String serverName = h.getServer().toString();
+        if (isOfflineTable != null) {
+          if (isOfflineTable) {
+            serverName += "_OFFLINE";
```

Comment:
please use one of the declared strings for "OFFLINE" and "REALTIME". Or, declare new shortened versions as static string (e.g. "_R" and "_O") and use them